IISCRUM-1328 python and requests to proxy

akononen opened the pull request.

akononen committed:
Update Dockerfile

Add python, pip and requests library installation to Dockerfile

akononen committed:
Update Dockerfile

No need fo separate pip installation

bulkware commented on Dockerfile:
```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
+    cd /usr/local/bin && \
+    ln -s /usr/bin/python3 python && \
```
I know this is only proxy, but are you sure you want to make Python 3 as a default interpreter? Are there any other scripts that use Python 2? The default interpreter can also be defined in the script's shebang line.

akononen replied:
To my knowledge there are currently no python scripts in proxy and this is the reason Python has to be installed there in the first place. I figured Python3 would be better choice as 2 is no longer supported. I can't think of a reason to choose Python2 instead of Python3.

bulkware replied:
Agreed.

sapajune commented on Dockerfile:
```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
+ 
+# Install Python, pip and requests library 
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
```
Does it really need python3-dev - shouldn't python3 be sufficient?

akononen replied:
Good point. I tested with python3 and it seems to be working. I will make this change.

sapajune commented on Dockerfile:
```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
+    cd /usr/local/bin && \
+    ln -s /usr/bin/python3 python && \
```
I think the installation could be included in the preceding layer without repeatedly running apt-get update.

akononen replied:
Everything is now included in the same layer

akononen committed:
Update Dockerfile

Put python installation in same RUN sequence as SQLite installation. Also change Python version.

akononen committed:
Update Dockerfile

Incorrect indentation fixed

akononen committed:
Update README.md
